src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -21,11 +21,13 @@ public class MsQuicTests : MsQuicTestBase
         [Fact]
         public async Task BasicTest()
         {
+            using QuicListener listener = CreateQuicListener();
```

Comment:
I think this is more of a style thing. I personally prefer initialization of the listener in the constructor, but I'm not strongly opposed to making it explicit